test/App/Fossa/Report/AttributionSpec.hs

```diff
@@ -0,0 +1,74 @@
+module App.Fossa.Report.AttributionSpec
+  ( spec,
+  )
+where
+
+import App.Fossa.Report.Attribution
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+import Prologue
+import Test.Hspec
+import Test.Hspec.Hedgehog
+
+defaultRange :: Range Int
+defaultRange = Range.linear 0 50
+
+genLicense :: Gen License
+genLicense =
+  License
+    <$> (LicenseName <$> arbitraryText)
+    <*> Gen.maybe (LicenseContents <$> arbitraryText)
+
+genProject :: Gen Project
+genProject = Project <$> arbitraryText <*> arbitraryText
+
+genDependency :: Gen Dependency
+genDependency =
+  Dependency
```

Comment:
I don't entirely understand what is happening here but it appears to be doing the correct thing. We should chat about this tomorrow because I'm interested to know what is happening!